### PR TITLE
Convert more packages to `types-2.0` style

### PR DESCRIPTION
**Pull request by ghost**
This means using `index.d.ts`, `<reference types="..." />` (or just import), and using external modules where possible.

### CODE REVIEW

File: bwip-js/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,83 @@
+// Type definitions for bwip-js 1.1.1
```
Comment (andy-ms) [inline bwip-js/index.d.ts]: Moved renames to their own commit. `git log --follow bwip-js/index.d.ts` should work now.

Comment (jramsay) [inline bwip-js/index.d.ts]: great - thanks!
